flow/project.py

```diff
@@ -1975,10 +1972,10 @@ def submit_operations(self, operations, _id=None, env=None, parallel=False, flag
             Show information about available template variables and filters and exit.
         :type show_template_help:
             bool
-        :param kwargs:
+        :param \*\*kwargs:
```

Comment:
The current formatting implies that we have a keyword argument literally named `kwargs`. However, it is difficult to document `*args` and `**kwargs` so that they appear correctly in the rendered documentation. We must use a raw string (`r"""` above) and escape the asterisks so that it will render as `**kwargs` in the docs as desired.